assets/master.js

```diff
@@ -104,6 +105,19 @@
       byId('stats').innerHTML = '<span class="failure">' + escapeHTML(err) + '</span>';
       byId('minify-btn').disabled = false;
     });
+    byId('copy-btn').onclick = function() {
+      let bgColorButton = byId('copy-btn').style.backgroundColor;
```

Comment:
Can / should be done with a class.

Reply:
I didn't get it, can you elaborate?

Reply:
Don't set (inline) styles with JS, set a class with JS and apply the needed styles via style tag. That's is better to maintain.

Reply:
Sure added it in!